build/build_app_engine_package.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
```

Comment:
nit: run gpylint
I don't know how strict we want to be with build files, but seems to be a good thing to get into a habit of running lint tools.